lib/linguist/languages.yml

```diff
@@ -2781,7 +2781,7 @@ JSONiq:
   language_id: 177
 Janet:
   type: programming
-  color: "#aa4400"
+  color: "#0886a5"
```

Comment:
@bakpakin Should this actually be #aa4400? The auburn seems more Janet-y (for want of a better term) than the blue? Or perhaps purple? That's #ccaaff.

Reply:
@Alhadis Oh, yes, sorry, I think your reversion makes complete sense. I left the comment because the PR is from a few years ago and perhaps @bakpakin's opinion has changed since then (I should have stated this in the comment but didn't). Definitely agree it's his decision.

Reply:
Purple is nice, but so is auburn, and I recall trying to avoid color collisions as well as near colors, which is something that linguist had a tool for dealing with last I remember.